deltacat/compute/converter/utils/converter_session_utils.py

```diff
@@ -121,3 +123,53 @@ def sort_data_files_maintaining_order(data_files: DataFileList) -> DataFileList:
     if data_files:
         data_files = sorted(data_files, key=lambda f: (f[0], f[1].file_path))
     return data_files
+
+
+class SnapshotType(Enum):
+    """Enumeration of possible snapshot types."""
+
+    NONE = "none"
+    APPEND = "append"
+    REPLACE = "replace"
+    DELETE = "delete"
+
+
+def _get_snapshot_description(
```

Comment:
Since this isn't describing the snapshot but instead the action that we'll take based on the snapshot, maybe something like:
```suggestion
def _get_snapshot_action_description(
```